MWSE/TES3DataHandler.h

```diff
@@ -64,7 +64,7 @@ namespace TES3 {
 		Iterator<GlobalVariable> * globals; // 0x38
 		Iterator<Dialogue> * dialogues; // 0x3C
 		Iterator<Region> * regions; // 0x40
-		Iterator<BaseObject> * birthsigns; // 0x44
+		Iterator<Birthsign> * birthsigns; // 0x44
```

Comment:
Until we transition to sol3 this won't work without some handholding. See the utter ~~horror~~ beauty of TES3CollectionsLua.cpp, and add a `bindGenericObjectIterator` call for birthsigns.